src/Microsoft.DocAsCode.Metadata.ManagedReference/ExtractMetadataWorker.cs

```diff
@@ -23,7 +23,10 @@ namespace Microsoft.DocAsCode.Metadata.ManagedReference
 
     public sealed class ExtractMetadataWorker : IDisposable
     {
-        private readonly Lazy<MSBuildWorkspace> _workspace = new Lazy<MSBuildWorkspace>(() => MSBuildWorkspace.Create());
+        private readonly Lazy<MSBuildWorkspace> _workspace = new Lazy<MSBuildWorkspace>(() => MSBuildWorkspace.Create(new Dictionary<string, string>
+        {
+            { "Configuration", "Release" }
```

Comment:
I agree with it. The aim of this PR is only to change the default behavior, and I'll provide metadata option in another PR if users need it.